### DISCUSSION TITLE
Mugglepay回调失败

### DISCUSSION THREAD

**Discussion by rikbyte** [Baiyuetribe/kamiFaka#76] [Bug反馈]
已设置web_url,Mugglepay回调失败,在Mugglepay后台手工补单也失败,付款后不跳回原站

**Comment by Baiyuetribe** [comment]
不可能，好多人用的好好的

**Reply by rikbyte** [reply]
我在你的演示站（http://103.40.247.22:8000/）设置了麻瓜包支付，我刚试了一下有一样的问题

**Reply by kaze-desu** [reply]
我这边也是同样的问题，无法正常回调

**Comment by wngx99** [comment]
同样的问题。
